src/tools/Tools.mjs

```diff
@@ -121,8 +129,12 @@ export default class Tools {
 
     static getSvgTexture(url, w, h) {
         let factory = (cb, stage) => {
-            this.createSvg(cb, stage, url, w, h);
-        }
+            if (Utils.isWeb) {
```

Comment:
Is this necessary? I can see only one implementation of createSvg, the one in platform is gone, isn't it?

Reply:
stage.platform.createSvg is now in the Lightning-Spark repo platform area.

Reply:
I'm not sure if I understood you correctly, but I don't see any files commited other that TextTextureRenderer.js in spark platform area.

Reply:
Currently, the files are located here: https://github.com/pxscene/Lightning-Spark

It's a separate repo similar to Lightning-node.